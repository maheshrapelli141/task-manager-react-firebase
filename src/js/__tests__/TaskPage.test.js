import React from 'react';
import { shallow } from 'enzyme';

import TaskPage from '../components/TaskPage';

const wrapper = shallow(<TaskPage />);

describe('TaskPage component', () => {
  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
