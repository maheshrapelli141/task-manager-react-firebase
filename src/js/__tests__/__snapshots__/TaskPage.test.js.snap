// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskPage component should render correctly 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row"
  >
    <div
      className="col-12 col-md-3"
    >
      <Connect(TaskForm) />
    </div>
    <div
      className="col-12 col-md-9"
    >
      <Connect(TaskList) />
    </div>
  </div>
</div>
`;
